store/src/db.rs

```diff
@@ -27,13 +34,42 @@ impl Db {
 
         info!(sqlite = format!("{}", config.sqlite.display()), "Connected to the DB");
 
+        // Feature used to support `IN` and `NOT IN` queries
+        conn.interact(|conn| array::load_module(conn))
+            .await
+            .map_err(|_| anyhow!("Loading carray module failed"))??;
+
         conn.interact(|conn| migrations::MIGRATIONS.to_latest(conn))
             .await
             .map_err(|_| anyhow!("Migration task failed with a panic"))??;
 
         Ok(Db { pool })
     }
 
+    /// Inserts a new nullifier to the DB.
+    ///
+    /// This method may be called multiple times with the same nullifier.
+    pub async fn add_nullifier(
```

Comment:
Is this method (and other similar methods) used primarily for testing purposes? If so, we should probably indicate this in the comments and maybe group them together in a "testing" section or something similar.

Eventually, there will be only a single method for updating the data in the store - `apply_block`. This method will lock the database for update and perform all required updates atomically.

Reply:
In this PR it was added primarily for testing purposes, I assume this would be used in the future for other endpoints too (e.g. apply_block can call these methods).

- Put the methods behind a `cfg(test)`